.github/pull_request_template.md

```diff
@@ -1,17 +1,9 @@
-# Why was this change made? 🤔
+# Why was this change made?
 
+<!-- Feel free to use whatever GitHub copy/pastes into the PR description above, or even just `Fixes #1234` if the PR is closely tied to an issue. -->
 
 
-# How was this change tested? 🤨
-
-⚡ ⚠ If this change has cross service impact (including writing to shared file systems or interacting with other SDR APIs), ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡
-
-⚡ ⚠ If this change updates the Argo UI, run all integration tests that use Argo and create a PR on the integration test repo to fix anything this change breaks. ⚡
-
-
-# Does your change introduce accessibility violations? 🩺
-
-⚡ ⚠ Please ensure this change does not introduce accessibility violations (at the WCAG A or AA conformance levels); if it does, include a rationale. See the [Infrastructure accessibility guide](https://github.com/sul-dlss/DeveloperPlaybook/blob/main/best-practices/infra-accessibility.md) for more detail. ⚡
-
+# How was this change tested?
 
+<!-- If this change has cross-service impact (including writing to shared file systems or interacting with other SDR APIs), or includes changes to the Argo UI, **please run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)** and/or test in [stage|qa] environment. -->
```

Comment:
Suggest removing "including writing to shared file systems or interacting with other SDR APIs".

I doubt anyone runs integration tests for most Argo UI changes.

Reply:
I don't think we can reduce noise further by removing the link to the integrations test repo since this is internal.

Otherwise, looks good to me.

Reply:
Historically, I think we're at best "middlin" at remembering to run integration tests.  I think we'll be similarly "poor" or "middlin" at running accessibility checks on UI changes.   I think a brief reminder to run integration tests when warranted and to do accessibility checking for UI changes is useful.

Reply:
How about 

```<!-- run infrastructure integration test(s) if cross-service impact, including writing to shared file systems, interacting with APIs of other apps, or Argo UI changes -->

Reply:
I'm going to try to merge all of the suggestions together. 🤞🏻